package.json

```diff
@@ -208,7 +208,7 @@
     "ts-node": "5.0.0",
     "tslint": "5.8.0",
     "tslint-config-prettier": "1.7.0",
-    "typescript": "2.6.2",
+    "typescript": "3.5.3",
```

Comment:
I think it's danger to upgrade a major version, because it's incompatible with current `webpack` 3.12.